WebUI/Data/Configurations/GameConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+﻿using ConestogaInsidersClub.Data.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace ConestogaInsidersClub.Data.Configurations
+{
+    public class GameConfiguration : IEntityTypeConfiguration<Game>
+    {
+        public void Configure(EntityTypeBuilder<Game> builder)
+        {
+            builder.Property(e => e.GameId);
```

Comment:
Why this one isn't the key like the others?

Reply:
cause he's a special one

Reply:
actually, any `<Entity>Id` is a key by convention, so let me clean that up